src/app/Components/Authorization/sign-up/sign-up.component.ts

```diff
@@ -36,18 +30,18 @@ export class SignUpComponent implements OnInit {
     this.service.form.reset();
   }
 
-  onSubmit(form: NgForm) {    
+  onSubmit(form: NgForm) {
```

Comment:
There aren't any use of form in this method.

Reply:
It's better to not remove unused input values because our project can crash.